com.core447.StreamController.yml

```diff
@@ -0,0 +1,105 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=session-bus # Allow plugins to connect to the session bus
+  - --device=all # Needed to communicate with the decks
+  - --talk-name=org.freedesktop.Flatpak # Allow plugins to run system commands
```

Comment:
why sandboxing then?

Reply:
this is not allowed by default.

Reply:
It's not just about sandboxing but also easy publishing to all distros. As I said the main app needs quite a lot of permissions so that plugins can integrate into the os, that is what makes them so powerful.

Reply:
What specific commands do you need to run? and where are these plugins?

Reply:
StreamController has a build-in plugin store that downloads and installs them. Devs can submit them to the [Store](https://github.com/StreamController/StreamController-Store). This allows everyone to install what they need or develop missing integrations.

The OS plugin has an action to run commands on the host that can be used to open apps, move windows, etc. This is why I need the spawn permissions for StreamController